Library/DiscUtils.Ext/VfsExtFileSystem.cs

```diff
@@ -73,13 +73,13 @@ public VfsExtFileSystem(Stream stream, FileSystemParameters parameters)
             long blockDescStart = (superblock.FirstDataBlock + 1) * (long)superblock.BlockSize;
 
             stream.Position = blockDescStart;
-            var bgDescSize = superblock.Has64Bit ? BlockGroup64.DescriptorSize64 : BlockGroup.DescriptorSize;
+            var bgDescSize = superblock.Has64Bit ? superblock.DescriptorSize : BlockGroup.DescriptorSize;
```

Comment:
Could this be `var bgDescSize = superblock.DescriptorSize;`?

Reply:
The superblock.DescriptorSize value is only valid with the SixtyFourBit feature. In my investigation, without this feature the value was often zero.